nautobot/dcim/elevations.py

```diff
@@ -36,14 +36,9 @@ def __init__(self, rack, user=None, include_images=True, base_url=None, display_
 
     @staticmethod
     def _get_device_description(device):
-        return "{} ({}) — {} ({}U) {} {}".format(
-            device.name,
-            device.device_role,
-            device.device_type.display,
-            device.device_type.u_height,
-            device.asset_tag or "",
-            device.serial or "",
-        )
+        asset_tag = device.asset_tag or ""
+        serial = device.serial or ""
+        return f"{device.name} ({device.device_role}) — {device.device_type.display} ({device.device_type.u_height}U) {asset_tag} {serial}"
```

Comment:
I'm on the fence about whether this one is an improvement in readability. I'd be tempted to leave it as-is.